### ISSUE TITLE
Is this applicable to 2012r2?  

### ISSUE THREAD

**Issue by alexknab** [MicrosoftDocs/windowsserverdocs#3031]
Does this article apply to 2012/2012r2?   The first paragraph indicates it does, whereas the "applies to" indicates 2016 only.   Please clarify, and if this no longer applies to 2012/2012r2, what does?  

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 03114469-107c-ca91-7f1f-f74be0701699
* Version Independent ID: ed0ee391-fdc8-4c22-4000-9ff103b33a17
* Content: [Virtualizing Domain Controllers using Hyper-V](https://docs.microsoft.com/en-us/windows-server/identity/ad-ds/get-started/virtual-dc/virtualized-domain-controllers-hyper-v#feedback)
* Content Source: [WindowsServerDocs/identity/ad-ds/get-started/virtual-dc/virtualized-domain-controllers-hyper-v.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/identity/ad-ds/get-started/virtual-dc/virtualized-domain-controllers-hyper-v.md)
* Product: **windows-server-threshold**
* GitHub Login: @MicrosoftGuyJFlo
* Microsoft Alias: **joflore**

**Comment by MicrosoftGuyJFlo** [comment]
@alexknab Yes the guidance is still applicable for previous versions. There are sections that specifically call out versions of support.
